textile/features.textile

```diff
@@ -1606,7 +1606,7 @@ h4. ClientOptions
 ** @(TO3d)@ @tls@ boolean - defaults to true. If false, will not use TLS for all connections
 ** @(TO3e)@ @autoConnect@ boolean - defaults to true. If false, suppresses the automatic initiation of a connection when the library is instantiated
 ** @(TO3f)@ @useBinaryProtocol@ boolean - defaults to true. If false, forces the library to use the JSON encoding for REST and Realtime operations, instead of the default binary msgpack encoding
-** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of messages when connection states that anticipate imminent connection (connecting and disconnected). Instead, publish and presence state changes will fail immediately if not in the connected state
+** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of @MESSAGE@ or @PRESENCE@ messages when connection is either in the `CONNECTING` or `DISCONNECTED` states. Otherwise those messages should be failed immediately if not in the `CONNECTED` state.
```

Comment:
`@MESSAGE@ or @PRESENCE@ messages` looks a bit odd, I think it's better `messages or presence messages` or `@Message@ or @PresenceMessage@`. Also I believe "`" should be replaced with "@".

```suggestion
** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of messages or presence messages when connection is either in the @CONNECTING@ or @DISCONNECTED@ states. Otherwise those messages should be failed immediately if not in the @CONNECTED@ state.
```

Reply:
a015fe14

Reply:
Replaced @ProtocolMessages@ with @ProtocolMessage@s